### DISCUSSION TITLE
[Feature] ML_URL envrionment

### DISCUSSION THREAD

**Discussion by jtagcat** [immich-app/immich#10169] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

`ML_URL` for Admin → Settings → Machine Learning Settings → URL.

Currently, when deploying, the connection is not made (unlike with Postgres and Redis), and has to be done manually.

I don't know where is the limit, I'd like to also see `EXTERNAL_DOMAIN`, and e-mail configuration (which define external connectivity to the container).

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
https://immich.app/docs/install/config-file
